Source/Core/Core/HW/WiimoteEmu/EmuSubroutines.cpp

```diff
@@ -258,7 +258,12 @@ void Wiimote::HandleRequestStatus(const OutputReportRequestStatus&)
 
 void Wiimote::HandleWriteData(const OutputReportWriteData& wd)
 {
-  // TODO: Are writes ignored during an active read request?
+  if (m_read_request.size)
+  {
+    // FYI: Writes during an active read will occasionally produce a "busy" (0x4) ack.
+    // We won't simulate that as it often does work. Poorly programmed games may rely on it.
+    WARN_LOG(WIIMOTE, "WriteData: write during active read request.");
```

Comment:
Is the risk big enough to warrant statistics gathering?

Reply:
Meh. Usually a real remote processes the write just fine in the case. Spamming writes during an active read fail about 22% of the time here. It seems to just be getting lucky with the timing. It shouldn't be a problem to just always work instead of working 78% of the time.